src/components/timeline/timeline.container.js

```diff
@@ -0,0 +1,12 @@
+import { connect } from 'react-redux';
+import * as TimelineActions from 'app/actions/timeline';
+import Timeline from './timeline';
+
+export default connect(
+	state => ( {
+		revisions: state.timeline
+	} ),
+	dispatch => ( {
+		fetchTimeline: () => dispatch( TimelineActions.fetch() )
```

Comment:
This action doesn't exist, since the component wont be dispatching anything, we can remove the second argument of `connect()`